app/components/menu.tsx

```diff
@@ -8,7 +8,7 @@ import { MenuItem } from "@/components/menu-item";
  */
 export function Menu() {
   return (
```

Comment:
The hidden-ness of this is now defined in the positioning for its containing HeaderNav component. Much cleaner and consistent.